clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/security/EcsCredentialsInitializer.java

```diff
@@ -78,11 +83,12 @@ public ECSCredentialsConfig ecsCredentialsConfig() {
 
             NetflixECSCredentials ecsCredentials =
                 new NetflixAssumeRoleEcsCredentials(
-                    (NetflixAssumeRoleAmazonCredentials) credentialsLoader.load(ecsCopy).get(0),
+                    (NetflixAssumeRoleAmazonCredentials)
```

Comment:
The loop needs to be changed to not use `AccountCredentials` but `NetflixAmazonCredentials` without cast.